### PR TITLE
Move range

### PR DESCRIPTION
**Pull request by marehr**
last header in `range/*` that needs to be moved :)

resolves https://github.com/seqan/product_backlog/issues/272

### CODE REVIEW

**Review by SGSSGene — APPROVED** [review]

File: include/seqan3/core/range/detail/adaptor_base.hpp [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    constexpr friend auto operator|(adaptor_base && me, arg_t && arg)
+    {
+        return combined_adaptor{static_cast<derived_type &&>(me), std::forward<arg_t>(arg)};
+    }
+};
+
+} // namespace seqan3::detail
+
+namespace seqan3::detail
+{
```
Comment (eseiler) [inline include/seqan3/core/range/detail/adaptor_base.hpp:204]:
```suggestion
```

File: include/seqan3/core/range/detail/adaptor_for_view_without_args.hpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ * \ingroup views
+ * \brief Provides seqan3::detail::adaptor_for_view_without_args
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/std/ranges>
+
+#include <seqan3/core/range/detail/adaptor_base.hpp>
```
Comment (eseiler) [inline include/seqan3/core/range/detail/adaptor_for_view_without_args.hpp:18]:
```suggestion

#include <seqan3/core/range/detail/adaptor_base.hpp>
```

File: include/seqan3/core/range/detail/adaptor_from_functor.hpp [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+/*!\file
+ * \ingroup views
+ * \brief Provides seqan3::detail::adaptor_from_functor
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/std/ranges>
```
Comment (eseiler) [inline include/seqan3/core/range/detail/adaptor_from_functor.hpp:16]:
```suggestion
```

File: include/seqan3/range/views/detail.hpp [hunk 4]
```diff
@@ -13,401 +13,9 @@
```
Comment (eseiler) [inline include/seqan3/range/views/detail.hpp:13]: ^^^ Add the deprecation notes to doxygen?

**Review by eseiler — CHANGES_REQUESTED** [review]
💅

**Review by eseiler — APPROVED** [review]

**Comment by eseiler** [comment]
rebase or squash?

**Comment by marehr** [comment]
> rebase or squash?

I'll rebase

**Pull request merged**